src/qibo/models/variational.py

```diff
@@ -301,6 +379,26 @@ def minimize(
         return best, params
 
 
+def _cvar_loss(params, qaoa, hamiltonian, state):
+    if state is not None:
+        state = hamiltonian.backend.cast(state, copy=True)
+    qaoa.set_parameters(params)
+    state = qaoa(state)
+    return cvar(hamiltonian, state)
+
+
+def _dummy_zero(params, qaoa, hamiltonian, state, mode):
```

Comment:
Do you really need this function?

Reply:
Thanks. not really, I was debugging and I forgot to remove it.